index.html

```diff
@@ -2610,6 +2619,161 @@ <h2>Security Considerations</h2>
 
   </section>
 
+  <section class="appendix">
+    <h4>Timecode-related metadata</h4>
+    <p><a>DAPT Documents</a> express time as <em>media time</em>,
+      which assumes that there is a reference start time (zero) on a media
+      timeline, and a fixed playrate.
+      An alternative scheme that is used in some workflows is to
+      synchronise media components using timecode,
+      time stamps that are applied, for example to each frame of video.
+    </p>
+    <p>Workflows that create <a>DAPT documents</a> from
+      resources that use timecode need to map the timecode
+      values onto the <a>DAPT document</a>'s timeline.</p>
+    <p>If this mapping is not correct, presentation of the
+      <a>DAPT Document</a> will not be synchronised with the <a>related media object</a>.
+      In order to achieve correct synchronisation in this scenario,
+      the timecode corresponding to the start of the programme needs to be known.</p>
```

Comment:
> the timecode corresponding to the start of the programme needs to be known.

This is too strong. Having one timecode is sufficient. It could be the first content timecode.  

```suggestion
       a timecode, such as the one corresponding to the start of the programme, needs to be known.</p>
```